examples/static_react_task_with_tips/webapp/src/css/style.css

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+body {
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
+    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
+  margin: 0;
+}
+
+.hero {
+  padding: 3rem 1.5rem;
+  background-color: #f5f5f5;
+  font-size: 1.25rem;
+}
+.section {
+  padding: 3rem 1.5rem;
+}
+.task-text {
+  font-size: 2rem;
+  font-weight: 600;
+  line-height: 1.125;
+  margin: 0 0 2rem 0;
+}
+
+.button-row {
+  display: flex;
+}
+
+.button {
+  height: 3rem;
+  padding: 2rem;
+  border: 0;
+  color: white;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  white-space: nowrap;
+
+  border-radius: 0.25rem;
+  margin-right: 1rem;
+  font-size: 1.7em;
+  cursor: pointer;
+}
+.good {
+  background-color: #48c774;
+}
+.bad {
+  background-color: #f14668;
+}
```

Comment:
I wrote custom styles for this task as the inherit styles from bulma & bootstrap were causing the tips popup to look weird.

Reply:
this is a larger issue though, we must not rely on or mandate users to have to update their styles to avoid conflicts. Ideally their is scoping and separation. Perhaps we should use the Shadow DOM? https://web.dev/shadowdom-v1/ See the scoped CSS section.

Reply:
So I got the shadow dom to work, but I am not sure that this is fully desired. I believe the point of the headless Tips component is to remove most of the styles and to allow the user to modify the styles so that they can style it themselves. This isn’t possible with the shadow dom as you can’t apply styles

Reply:
how about we toggle it based on the headless prop? if they specify headless, it doesn't use ShadowDOM, if they don't then it uses ShadowDOM?

Reply:
Yeah, I'll definitely do that, but it doesn't change the underlying reason why I added these styles. The style conflicts with bulma were making the headless version of the component look really ugly. I removed bulma and added these styles to make the task look as similar to the static_react_task example

Reply:
This is how the headless component looks when the shadow dom is off:
<img width="587" alt="Screen Shot 2022-07-12 at 10 16 29 AM" src="https://user-images.githubusercontent.com/55665282/178511673-1400deb8-6eed-4905-85bb-1c83595ce8da.png">

The styles from bulma removes bold from headers & keeps their size the same as regular text. I added the above styling and removed bulma so that the rest of the task looked the same as the static_react_task and so that the headless version of tips looked normal.